data.sql

```diff
@@ -10,3 +10,13 @@ INSERT INTO animals VALUES (
 INSERT INTO animals VALUES (
 4,'Devimon', '2017-05-12', 5, TRUE, 11
 );
+
+-- Vet clinic database: query and update animals table
+INSERT INTO animals(id, name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES
+(5, 'Charmander', '2020-02-8', 0, FALSE, -11),
+(6, 'Plantmon', '2021-11-15', 2, TRUE , -5.7),
+(7, 'Squirtle', '1993-04-02', 3, FALSE, -12.13),
+(8, 'Angemon', '2005-06-12', 0, TRUE, -45),
+(9, 'Boarmon', '2005-07-07', 7, TRUE, 20.4),

```

Comment:
- [ ] As mentioned by the previous :point_up: reviewer based on the data you inserted into your database, it appears that the number of animals who have never tried to escape is 2. The data you inserted is not correct if you check the requirement the `Angemon` has tried to escape once but in your case, you put zero instead of 1 so kindly fix this issue as well :slightly_smiling_face: 

![Screenshot from 2023-09-22 19-46-39](https://github.com/MargaretTechworld/Vet-Clinic-Database/assets/45226848/a231bc98-022e-416d-b8c9-93d4ca182e43)

Reply:
Hi @aamirkhan2478,
I have made the necessary corrections.
Thank you
<img width="607" alt="CORRECTIONS" src="https://github.com/MargaretTechworld/Vet-Clinic-Database/assets/60655304/0a8dd301-e490-4931-ac6f-3d067c8981d9">

<img width="930" alt="Correction 2" src="https://github.com/MargaretTechworld/Vet-Clinic-Database/assets/60655304/b8bc301e-c325-4d74-821d-8795d5831b02">